src/main/scala/beam/analysis/AnalysisProcessor.scala

```diff
@@ -0,0 +1,42 @@
+package beam.analysis
+
+import com.zaxxer.nuprocess.{NuAbstractProcessHandler, NuProcess, NuProcessBuilder}
+import java.nio.ByteBuffer
+import java.nio.charset.StandardCharsets
+
+import beam.utils.logging.ExponentialLazyLogging
+
+object AnalysisProcessor extends ExponentialLazyLogging {
+
+  def iterationEndBulkAnalysisOutput_Async(currentIterationNumber: Int) = {
+    fireAndForgetPythonScript("src/main/python/events_analysis/drive_to_transit.py")
```

Comment:
Need to pass in the arg so the script can know what events file to handle.

Reply:
Done